apparmor.d/groups/virt/containerd

```diff
@@ -36,38 +37,44 @@ profile containerd @{exec_path} flags=(attach_disconnected) {
   umount @{run}/netns/cni-@{uuid},
 
   signal (receive) set=term peer=dockerd,
+  signal (send) set=kill peer=cni-calico,
 
-  @{exec_path}                        mr,
-  /{usr/,}{s,}bin/apparmor_parser     rPx,
+  @{exec_path} mr,
+  /{usr/,}{s,}bin/apparmor_parser rPx,
   /{usr/,}bin/containerd-shim-runc-v2 rPUx,
-  /{usr/,}bin/kmod                    rPx,
-  /{usr/,}bin/unpigz                  rPUx,
-  /{usr/,}{local/,}{s,}bin/zfs        rPx,
+  /{usr/,}bin/kmod rPx,
+  /{usr/,}bin/unpigz rPUx,
+  /{usr/,}{local/,}{s,}bin/zfs rPx,
 
-  /etc/cni/              rw,
-  /etc/cni/{,**}         r,
-  /etc/cni/net.d/        rw,
+  / r,
+
+  /etc/cni/ rw,
+  /etc/cni/{,**} r,
+  /etc/cni/net.d/ rw,
   /etc/containerd/*.toml r,
 
-  /opt/cni/bin/loopback  rPx,
-  /opt/cni/bin/portmap   rPx,
+  /opt/cni/bin/loopback rPx,
+  /opt/cni/bin/portmap rPx,
   /opt/cni/bin/bandwidth rPx,
-  /opt/cni/bin/calico    rPx,
+  /opt/cni/bin/calico rPx,
 
   /opt/containerd/{,**} rw,
 
+  /var/lib/cni/{,**/} w,
   /var/lib/cni/results/cni-loopback-@{uuid}-lo wl,
+  /var/lib/cni/results/cni-loopback-[0-9a-z]*-lo wl,
+  /var/lib/cni/results/k8s-pod-network-[0-9a-z]*-eth0 wl,
```

Comment:
As the interfaces can have a large set of name, you cannot just use `eth0` here.

Reply:
These interface names are created and managed by Calico. I haven't found any documentation yet on whether these names are adjustable, or different on systems where eth* is used by the host. Calico does have the capability to manage host interfaces as well. it just so happens to be used by Kubernetes so much as an CNI. But I don't really know what other interface names I can expect.

Perhaps something like this could work:
`/var/lib/cni/results/k8s-pod-network-[0-9a-z]*-* wl,`